src/main/java/survey/NumberQuestion.java

```diff
@@ -0,0 +1,26 @@
+package survey;
+
+import javax.persistence.Entity;
+import java.util.ArrayList;
+
+@Entity
+public class NumberQuestion extends Question {
+
+    private ArrayList<String> answers = new ArrayList<>();
+
+    /**
+     * Default constructor
+     */
+    public NumberQuestion() {
+        super();
```

Comment:
I believe a default constructor just calling super is implemented by default in Java and therefore unnecessary

Reply:
@TaylorAbraham The spring stuff for repository databases gets mad if you don't have the default constructor